experimental/dds/tree2/src/feature-libraries/sequence-field/sequenceFieldEditor.ts

```diff
@@ -91,21 +94,59 @@ export const sequenceFieldEditor = {
 		count: number,
 		destIndex: number,
 		id: ChangesetLocalId,
-	): [moveOut: Changeset<never>, moveIn: Changeset<never>] {
+	): Changeset<never> {
+		const moveIn: Mark<never> = {
+			type: "MoveIn",
+			id,
+			count,
+			cellId: { localId: id },
+		};
 		const moveOut: Mark<never> = {
 			type: "MoveOut",
 			id,
 			count,
 		};
+		const indexAfterMoveOut = sourceIndex + count;
```

Comment:
This name confused me as I interpreted "after" temporally rather than spatially.